### ISSUE TITLE
CMKD应该属于哪一类单目检测器？

### ISSUE THREAD

**Issue by Senwang98** [Cc-Hy/CMKD#39]
@Cc-Hy 
hello
![image](https://user-images.githubusercontent.com/23122139/235298436-0c32628a-95bc-4dcb-bca8-6583ee2af12f.png)
CMKD提及的`depth pre-trained backbone`以及`initialize the backbone with the weights pre-trained on COCO`分别是指在什么阶段做的处理呢？不是太懂

**Comment by Senwang98** [comment]
@Cc-Hy 
用coco初始化resnet这个好理解，请问这里用depth pre-trained backbone是指？

**Comment by Cc-Hy** [comment]
Hi, depth pre-trained backbone means the image backbone is pre-trained on a certain dataset or split with depth loss only. You can refer to the experiment setting part for detail.

**Comment by Senwang98** [comment]
@Cc-Hy 
训练CMKD-BEV时，加载的是coco预训练权重。仅从config文件中根本看不出来depth-pretrained的含义，因为CMKD在这个仓库的训练完全没有仅训练depth loss的过程。实在有点费解，因为pretrain depth对性能影响是最大的，相较于蒸馏来说

**Comment by Cc-Hy** [comment]
Hi, @Senwang98
Pre-training brings performance improvements in two ways, the increased amount of data and distillation. Because kitti dataset is small, the improvement of additional data will be particularly obvious. And when with sufficient data and keeping the amount of data same, distillation performs much better than depth supervision, e.g., the comparisions on eigen clean, Nuscenes, or Waymo. So the BEV pre-training we use now is actually a replacement for depth pre-training.

**Comment by Senwang98** [comment]
@Cc-Hy 
感谢回复，回到问题本身的话，想请教一下，CMKD除了使用了COCO预训练权重的trick之外，是否类似DD3D一样对骨干预训练过（只要不是KITTI 7481张图，理论上都算使用了额外数据）。因为这关系到后续论文是否需要跟CMKD公平比较。

**Comment by Cc-Hy** [comment]
@Senwang98 
Yes, the experiments on kitti test set use additional unlabeled data from eigen clean split or kitti raw set.

**Comment by Senwang98** [comment]
@Cc-Hy 
感谢回复，是我的问题导致交流受阻。我没搞清楚eigen clean的含义，所以CMKD还是跟DD3D一样，先用KITTI深度估计的数据pretrain过。因此CMKD无论是16.99mAP还是半监督18.69mAP，均是使用了额外数据的。
